pkgs/development/python-modules/arelle/default.nix

```diff
@@ -16,30 +15,37 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "Arelle";
     repo = "Arelle";
-    rev = "cb24e35d57b562a864ae3dd4542c4d9fcf3865fe";
-    sha256 = "1sbvhb3xlfnyvf1xj9dxwpcrfiaf7ikkdwvvap7aaxfxgiz85ip2";
+    rev = "edgr${version}";
+    sha256 = "12a94ipdp6xalqyds7rcp6cjwps6fbj3byigzfy403hlqc9n1g33";
   };
   outputs = ["out" "doc"];
   patches = [
     ./tests.patch
   ];
   postPatch = "rm testParser2.py";
   buildInputs = [
-    sphinx_1_2
+    sphinx
     pytest
     py3to2
   ];
   propagatedBuildInputs = [
     lxml
     isodate
     numpy
+    openpyxl
   ] ++ lib.optional gui [
     tkinter
   ];
 
   # arelle-gui is useless without gui dependencies, so delete it when !gui.
   postInstall = lib.optionalString (!gui) ''
     find $out/bin -name "*arelle-gui*" -delete
+  '' +
+  # By default, not the entirety of the src dir is copied. This means we don't
+  # copy the `images` dir, which is needed for the gui version.
+  lib.optionalString (gui) ''
+    targetDir=$out/${python.sitePackages}
+    cp -vr $src/arelle $targetDir
```

Comment:
`-v` is probably not necessary, is it?

Reply:
No, but it does give extra debug output, which might be useful